tools/misc/prepare_framework_v2.py

```
@@ -140,21 +199,99 @@ def create_excel_from_yaml(yaml_path, output_excel=None):
+
+        # Fill localized columns with translated data
+        for rid, langs_data in lang_impl_groups_map.items():
+            if rid in ref_id_to_row:
+                row_num = ref_id_to_row[rid]
+                for lang_code, texts in langs_data.items():
+                    name_col = col_index_map.get(f"name[{lang_code}]")
+                    desc_col = col_index_map.get(f"description[{lang_code}]")
+                    if name_col:
+                        impl_content_sheet.cell(row=row_num, column=name_col, value=texts["name"])
+                    if desc_col:
+                        impl_content_sheet.cell(row=row_num, column=desc_col, value=texts["description"])
```

Pylint (3.3.7)

[refactor] Too many nested blocks (6/5)

(R1702)